src/snapshot.ts

```diff
@@ -85,7 +85,42 @@ export function absoluteToStylesheet(cssText: string, href: string): string {
   });
 }
 
+function getAbsoluteSrcsetString(doc: Document, attributeValue: string) {
+  if(attributeValue.trim() === "") {
+    return attributeValue
+  }
+  let resultingSrcsetString = ""
+
+  // srcset attributes is defined as such:
+  // srcset = "url size,url1 size1"
+  const srcsetValues = attributeValue.split(",")
+  srcsetValues.forEach((srcItem, i) => {
+    // removing all but middle spaces
+    const trimmedSrcItem = srcItem.trimLeft().trimRight()
```

Comment:
what's the difference between this and just `trim()`?

Reply:
Trim would also remove the middle spaces where srcset specifies [url] [size] which would result in illegible urls. trimLeft and trimRight keep the middle spaces.